etcdserver/v3_server.go

```diff
@@ -775,6 +778,7 @@ func (s *EtcdServer) linearizableReadLoop() {
 		if !done {
 			continue
 		}
+		trace.Step("read index received")
```

Comment:
can we add following fields to improve observability?
trace.AddField(traceutil.Field{Key: "appliedIndex", Value: s.getAppliedIndex})
trace.AddField(traceutil.Field{Key: "readStateIndex", Value: rs.Index})

Reply:
ci is failed. https://travis-ci.com/github/etcd-io/etcd/jobs/391026189